README_CHS.md

```
@@ -0,0 +1,218 @@
+<div align="center">
+
+<img src="assets/godoxy.png" width="200">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)